scripts/createDB.sql

```diff
@@ -151,8 +174,53 @@ CREATE TABLE `soen_390_db`.`ordered_good` (
     
 	collate = utf8mb4_unicode_ci;
 
+-- Table to store the various finished goods that are associated with each customer order.
+CREATE TABLE `soen_390_db`.`customer_ordered_good` (
+  `orderId` INT NOT NULL,
+  `compositeId` INT NOT NULL,
```

Comment:
Why is this ``compositeId`` and not just ``goodId`` or ``finishedGoodId``?

Reply:
No reasons other than the column matching the manufacturing order